File: helloWorld.js
console.log("Hello world!", "I am Yann");

const name = "Katsu";
let age = 26;

console.log(`${name} is ${age} years-old.`);

age += 1;
console.log(`${name} is ${age} years-old.`);


const peeps = ['Yun', 'Julien', 'Munekatsu', 'Shogo']

// CRUD

// Create
peeps.push("Grant");
console.log(peeps);

// Read
console.log(peeps[0]);

// Update
peeps[0] = "Super Yun"
console.log(peeps);

// Delete
peeps.splice(2,1)
console.log(peeps);

peeps.forEach((person) => {
  console.log(person.toUpperCase());
});

age = 18;

if (age === 18) {
  console.log('Giri giri okay');
}
else if (age > 18) {
  console.log('You can vote');
}
else {
  console.log('Please get back to your toys.');
}


const student = {
  firstName: "Boris",
  lastName: "Paillard"
};

console.log(student);

console.log(student.firstName);

// A function that is squaring a number
const square = (num) => {
  return num * num
};

console.log(square(42));

const capitalize = (word) => {
  const firstLetter = word[0].toUpperCase();
  const remainder = word.substring(1).toLowerCase();
  return `${firstLetter}${remainder}`;
};


console.log(capitalize("heLlo"));

File: hello_world.rb
puts "Hello world!"

name = "Katsu"
age = 26
puts "#{name} is #{age} years-old."

age += 1
puts "#{name} is #{age} years-old."

peeps = ['Yun', 'Julien', 'Munekatsu', 'Shogo']

# CRUD
# Create
peeps << "Grant"
peeps.push("Lukasz")
p peeps

# Read
puts peeps[0]

# Update
peeps[0] = "Super Yun"
p peeps

# Delete
peeps.delete_at(2)
p peeps


peeps.each do |person|
  puts person.upcase
end

age = 13

if age == 18
  puts 'Giri giri okay'
elsif age > 18
  puts 'You can vote'
else
  puts 'Please get back to your toys.'
end


student = {
  firstName: "Boris",
  lastName: "Paillard"
}

p student


puts student[:firstName]

# A method that is squaring a number
def square(num)
  num * num
end


puts square(42)






